assets/lucashifacts

```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
 lucashi once dated an ex-convict
 lucashi is a fully licensed motorcycle rider in Ontario
 lucashi has climbed to the peak of mt. Elbert the tallest moutain in the rocky mountains
+the only bone lucashi has ever broken is his wrist bone
+lucashi has trained new employees at CoreAVI
+lucashi has a tendency to have drinks poured all over his pants
+lucashi got so drunk at a work party once, he forgot his underwear in his coworker's car
```

Comment:
I think it reads a bit nicer if it's like: lucashi once got so drunk at a work party that he forgot..